client/extensions/woocommerce/state/data-layer/index.js

```diff
@@ -14,6 +14,7 @@ import settingsGeneral from '../sites/settings/general/handlers';
 import shippingZoneLocations from './shipping-zone-locations';
 import shippingZoneMethods from './shipping-zone-methods';
 import shippingZones from './shipping-zones';
+import stripeConnectAccount from '../sites/settings/stripeConnectAccount/handlers';
```

Comment:
The folder name should probably be `stripe-connect-account`.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/17922/commits/4d6f76af0a519bc985494306ee71ecb57b15a373